zelastic.gemspec

```diff
@@ -25,10 +27,10 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'activesupport'
 
   spec.add_development_dependency 'activerecord'
```

Comment:
🤔 development dependency only? this is weird

Reply:
yeah, the dependencies are a bit strange, but didn't want to change it too much. 🤷 Even stranger is the elasticsearch gem being development only. I considered raising it to a proper dependency, but can be done later. No release yet in this PR, so I may do some more changes and do a proper v1 release, since it is still on v0.